src/Persistence/PaYaml/Models/SchemaV3/DataSourceInstance.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.PaYaml.Models.SchemaV3;
+
+public record DataSourceInstance
+{
+    public required string Type { get; init; }
```

Comment:
Since we know this will be a small well-known set, I would recommend that we make this an Enum type.